pkg/app/app.go

```diff
@@ -76,6 +76,10 @@ func ValidateFlags(bundle string, args []string, cmd *cobra.Command, opt *kobjec
 	replicationController := cmd.Flags().Lookup("replication-controller").Changed
 	deployment := cmd.Flags().Lookup("deployment").Changed
 
+	// Get the controller
+	controller := cmd.Flags().Lookup("controller").Value.String()
```

Comment:
I thought I put my comment on this, but no need to do a look up. Just use `opt`. `controller := opt.Controller`